Classes/Domain/Model/Cart/Cart.php

```diff
@@ -1280,7 +1280,8 @@ private function calcTax()
      */
     public function reCalc()
     {
-        $this->calcGross();
+	    $this->calcCount();
```

Comment:
I need shipping costs to be calculated per Item. After updating the amount in cart, i noticed that all the calculation was wrong. (taxes, shipping costs ...)